### DISCUSSION TITLE
Standardization with Tile Card

### DISCUSSION THREAD

**Discussion by jazzyisj** [nicufarmache/lovelace-big-slider-card#24] [Ideas]
What a great card!   Makes you wonder why the UI dev's didn't do this to the tile card, so intuitive! Thank you!  Looking forward to seeing if you can extend it to other entities (I see a few requests for that).

Would you consider standardizing some of the UI of the big slider card with the hass tile card?  I'd like to be able to use them interchangeably in my UI but the inconsistencies are bothersome.

1. Card background should really have been the standard card background color.  (ha-card-background).  I was able to accomplish this for myself with card_mod, but I think it should come out of the box that way for consistency, allowing the user to customize if desired.

2. Card size - the big slider card height is slightly smaller than a tile card and it looks terrible in a grid when used together.  Haven't been able to figure this one out with card_mod yet.

3. Entity name should not word wrap -> it pushes the secondary text off the card, becomes unreadable, and looks terrible.

4. Icon alignment - the big slider icon has less padding to the left than the tile card icon.

<img width="315" alt="image" src="https://user-images.githubusercontent.com/44064559/232487647-faea1838-359b-4493-901b-cb4f2485e3ba.png">

5. A halo around the entity icon would be great to add a little contrast between the icon and the background when entity is on and for a consistent look with the tile card.  Especially if one uses the state-light-active-color for the slider color.  Also, the icon when no rgb support should also be state-light-active-color, not white for consistency with other hass cards.

<img width="199" alt="image" src="https://user-images.githubusercontent.com/44064559/232492005-26436b57-2dfe-4b77-9dd5-5d65bbc38c2d.png">

6. Personally I would have used the state-light-active-color as the standard active slider color instead of the primary color for consistency with other cards.   Of course easily remedied with the color: option of the slider card, it's just a thought.

<img width="392" alt="image" src="https://user-images.githubusercontent.com/44064559/232491478-51939e2d-d9f5-4bde-a0bd-8eeb4fa9dc5d.png">

7. This is more of a functionality request.  I have several timed lights in my home.  Currently I display any active light timers in a separate card (see screenshot).  I'd love to be able to put those timers in the secondary text slot somehow.

<img width="166" alt="Screenshot 2023-04-17 090959" src="https://user-images.githubusercontent.com/44064559/232494627-377f8de1-16d6-4e4a-aa9e-46f1c5b742ec.png">

Thanks again for the great card.  It really gave my lighting dashboard a fresh new look!

<img width="1269" alt="image" src="https://user-images.githubusercontent.com/44064559/232495075-b124408e-caa7-43f3-b840-88ff5f083e37.png">

**Comment by nicufarmache** [comment]
Thanks for the ideas. I've been a little busy lately but i'll definitely try to make it more consistent with the rest of HA in the next verions
